package.json

```diff
@@ -69,7 +70,7 @@
     "angular": "1.x"
   },
   "dependencies": {
-    "babel-runtime": "^6.25.0",
+    "babel-runtime": "^6.26.0",
```

Comment:
@Ardeshir81 We use [`greenkeeper`](https://greenkeeper.io) to keep dependencies up-to-date. It will update `babel-runtime` for us.

Could you please remove bump of `babel-runtime` version from commit?